src/main/java/com/code_intelligence/jazzer/junit/RegressionTestArgumentProvider.java

```diff
@@ -39,47 +39,70 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.stream.Stream;
 import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.ExtensionContext.Namespace;
 import org.junit.jupiter.params.provider.Arguments;
 import org.junit.jupiter.params.provider.ArgumentsProvider;
 import org.junit.jupiter.params.support.AnnotationConsumer;
 
-class RegressionTestArgumentProvider implements ArgumentsProvider, AnnotationConsumer<FuzzTest> {
+class FuzzTestArgumentsProvider implements ArgumentsProvider, AnnotationConsumer<FuzzTest> {
```

Comment:
Something went wrong here. The name change and the new file suggest that this one should be removed, right?
Also `RegressionTestExtension` and `JazzerTestEngine`.

Reply:
I wonder how the build / tests worked in the CI ...

Reply:
This happened when I rebased on the merge of `//driver/...` with `//agent/...`. The files weren't referenced anywhere, which is why CI didn't complain.

Reply:
Much nicer now, thx! :smile: